app/models/user.rb

```diff
@@ -4,4 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   has_many :trainings
+
+  def full_name
+    return ("#{first_name} #{second_name}") if (first_name || second_name)
```

Comment:
it shouldn't be in the model, please move it to a helper for now and we'll introduce a decorator tomorrow

Reply:
done